graphene_linked_events/schema.py

```diff
@@ -800,7 +812,19 @@ def mutate(root, info, **kwargs):
         event_obj = json2obj(format_response(result))
         if result.status_code == 201:
             # Create palvelutarjotin event if event created successful
-            AddEventMutation._create_p_event(event_obj, organisation, p_event_data)
+            try:
+                AddEventMutation._create_p_event(event_obj, organisation, p_event_data)
+            except Exception as e:
+                logger.error(
```

Comment:
This could also be `logger.exception` which I think should automatically put the exception information into to the log.